tfexec/workspace_show.go

```diff
@@ -36,5 +36,5 @@ func (tf *Tofu) workspaceShowCmd(ctx context.Context) (*exec.Cmd, error) {
 		return nil, fmt.Errorf("workspace show was first introduced in Terraform 0.10.0: %w", err)
```

Comment:
From a simple `grep` into the branch, I can see a lot more mentions that could be reworked. Is this planned for later if ever?
![Screenshot 2025-05-12 at 07 45 07](https://github.com/user-attachments/assets/feb8d6cd-527a-4773-bd94-50e6d26aa173)